index.html

```diff
@@ -141,31 +144,33 @@ <h2>About Myself</h2>
             </div>
         </div>
         <br>
-        <ul class="sort">
-            <li class="lang">Language <img src="images/down.png" alt="down" class="arrow">
-                <ul>
-                    <li class="langu"> <img src="images/js.png" alt="js"> javascript</li>
-                    <li class="langu"> <img src="images/html.png" alt="html"> HTML</li>
-                    <li class="langu"> <img src="images/css.png" alt="css"> css</li>
-                </ul>
-            </li>
-        </ul>
-        <ul class="sort">
-            <li class="lang">Framworks <img src="images/side.png" alt="side" class="arrow1"></li>
-            <li class="lang">Skills <img src="images/side.png" alt="side" class="arrow1"></li>
-        </ul>
+        <div class="sort">
+            <ul>
+                <li class="lang"> <span>Language</span>  <img src="images/down.png" alt="down" class="arrow">
+                    <ul class="langu1">
+                        <li class="langu"> <img src="images/js.png" alt="js"> javascript</li>
+                        <li class="langu"> <img src="images/html.png" alt="html"> html</li>
+                        <li class="langu"> <img src="images/css.png" alt="css"> css</li>
+                    </ul>
+                </li>
+                <li class="lang1">Framworks <img src="images/side.png" alt="side" class="arrow1"></li>
+                <li class="lang1">Skills <img src="images/side.png" alt="side" class="arrow1"></li>
```

Comment:
- [ ] `optional`
- [ ] Kindly make the icons as the figma template please the screenshots below :v: 
- `your template`
![Screenshot from 2022-07-07 10-22-14](https://user-images.githubusercontent.com/66230306/177876343-33fef244-6758-4fb8-bb36-240371389447.png)

-`figma template`
![Screenshot from 2022-07-07 10-22-57](https://user-images.githubusercontent.com/66230306/177876708-8eecefc3-486c-4907-8f98-7a5dceb52849.png)